Testscripts/Linux/SetupRDMA.sh

```diff
@@ -54,15 +54,20 @@ function Verify_Result {
 function Main() {
 	LogMsg "Starting RDMA required packages and software setup in VM"
 	update_repos
-
 	# Install common packages
-	install_package "gcc zip"
+	install_package "gcc git zip"
+	# Change memory limits
+	echo "* soft memlock unlimited" >> /etc/security/limits.conf
```

Comment:
Please make sure this path is only available a certain distro. Should be conditional.

Reply:
I verified and it's available on all distros: sles/ubuntu/rhel/centos